src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageCreateActionImpl.java

```diff
@@ -0,0 +1,186 @@
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.internal.requests.restaction;
+
+import net.dv8tion.jda.api.entities.GuildChannel;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageChannel;
+import net.dv8tion.jda.api.entities.sticker.GuildSticker;
+import net.dv8tion.jda.api.entities.sticker.StickerSnowflake;
+import net.dv8tion.jda.api.requests.Request;
+import net.dv8tion.jda.api.requests.Response;
+import net.dv8tion.jda.api.requests.restaction.MessageCreateAction;
+import net.dv8tion.jda.api.utils.AttachedFile;
+import net.dv8tion.jda.api.utils.data.DataObject;
+import net.dv8tion.jda.api.utils.messages.MessageCreateBuilder;
+import net.dv8tion.jda.api.utils.messages.MessageCreateData;
+import net.dv8tion.jda.internal.requests.RestActionImpl;
+import net.dv8tion.jda.internal.requests.Route;
+import net.dv8tion.jda.internal.utils.Checks;
+import net.dv8tion.jda.internal.utils.message.MessageCreateBuilderMixin;
+import okhttp3.RequestBody;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.BooleanSupplier;
+import java.util.stream.Collectors;
+
+public class MessageCreateActionImpl extends RestActionImpl<Message> implements MessageCreateAction, MessageCreateBuilderMixin<MessageCreateAction>
+{
+    protected static boolean defaultFailOnInvalidReply = false;
+
+    private final MessageChannel channel;
+    private final MessageCreateBuilder builder = new MessageCreateBuilder();
+    private final List<String> stickers = new ArrayList<>();
+    private String nonce;
+    private String messageReferenceId;
+    private boolean failOnInvalidReply = defaultFailOnInvalidReply;
+
+    public static void setDefaultFailOnInvalidReply(boolean fail)
+    {
+        defaultFailOnInvalidReply = fail;
+    }
+
+    public MessageCreateActionImpl(MessageChannel channel)
+    {
+        super(channel.getJDA(), Route.Messages.SEND_MESSAGE.compile(channel.getId()));
+        this.channel = channel;
+    }
+
+
+    @Override
+    public MessageCreateBuilder getBuilder()
+    {
+        return builder;
+    }
+
+    @Override
+    protected RequestBody finalizeData()
+    {
+        if (builder.isEmpty())
+        {
+            if (!stickers.isEmpty())
+                return getRequestBody(DataObject.empty().put("sticker_ids", stickers));
+            throw new IllegalStateException("Cannot build empty messages! Must provide at least one of: content, embed, file, or stickers");
+        }
```

Comment:
Because you can only use stickers on the create message endpoint, see https://github.com/discord/discord-api-docs/discussions/4441

Reply:
Discussed internally bringing this to other endpoints; Sounds like it wasn't done only because it was forgotten; As far as I can tell there are no real reasons to block this.

I've added it to my infinitely long backlog.